meta-refkit/conf/distro/include/refkit-ci.inc

```diff
@@ -49,6 +49,11 @@ require conf/distro/include/refkit-development.inc
 REFKIT_VM_IMAGE_TYPES = "wic.xz wic.zip wic.bmap wic.xz.sha256sum"
 
 #
+# pre/post-build oe-selftests started by CI builder, whitespace-separated.
```

Comment:
Can you add a note here explaining that refkit-ci.inc itself is not active while running the selftests and why?